elasticsearch/Dockerfile

```diff
@@ -8,17 +8,19 @@ USER 0
 
 ENV HOME=/opt/app-root/src \
   JAVA_VER=1.8.0 \
-  ES_VER=1.5.2 \
-  ES_CONF=/usr/share/elasticsearch/config/elasticsearch.yml \
+  ES_VER=2.3.3 \
+  ES_CONF=/usr/share/elasticsearch/config \
```

Comment:
it should only be `/usr/share/elasticsearch/config`
in fact the /etc/elasticsearch line in the dockerfile is commented out... i'll clean it up